### ISSUE TITLE
Incorrect detection

### ISSUE THREAD

**Issue by Iwouldliketobeapig** [vuejs/eslint-plugin-vue#429]
eslint version: 4.15.0
eslint-plugin-vue: 4.3.0
node version: 8.9.0

```vue
<div class="details_header">
      <div>通知消息>消息详情</div>
      <div
      class="goback"
      @click="goBack"><返回</div>
</div>
```

![test](https://user-images.githubusercontent.com/20534862/37267948-4b2911ee-25fe-11e8-86bd-645369c613ce.png)

when i use '<' in the innerHtml,there will be wrong that tell me this is a in invalid tag name

**Comment by BHerbulot** [comment]
Same problem here with: 
eslint 4.18.1
eslint-plugin-vue 4.3.0
node 8.9.4

![capture d ecran 2018-03-13 a 14 00 43](https://user-images.githubusercontent.com/16306338/37343210-22959d82-26c7-11e8-8e8b-76e04a4b57b0.png)

**Comment by michalsnik** [comment]
Thank you for posting this issue @Iwouldliketobeapig I reproduced it [here](https://mysticatea.github.io/vue-eslint-demo/#eJyNVk2K5jYQvUrj9ScasmyaQAjkAskyG9ku25qWJU9J+vrzDHOL7AK5Sk4TyDHyJNmWf3ogi24+vVdSqapelfy1amxL1Uv16mmctPT04+/m6em1Vff04+nJk/P51+u/f//xz59/Jf45G7w+77Zh6RpWk0/E9rO6VRw0uerlK5wZ5zk03rJwYSKuXn64VcaKRjoSLTVasvQKVhuhpXMCf6o3G2bHSTIJQ734QmwLbtqLqXFW037pPzDx0vi0XUXnO8az1YKpp8cKtlSHvi83b0mTJ3GXBUFgQnLvDkAOZKSxJj4ybzTvAa0a5DNlZEWRYz+LZkByGk+czzqSk/RgzAHcFo9TyPTwLEVtkRlpoid/ZByNakU6qbUf2IZ+2KBgmtOJvba11CdQGYPLflRVZe5SqzZndtpQxjrAWLwPCrqbZLOlYFQPakWCIAfU2ct6O87Qu3DzWFu9Irb+hLi03kxs4+XGMuVLUQFwj+X0FXOku1NAMGAXa8uylMwPyomaOgtFHjQdTEtdWSBQQvVaMQbtlVaGCsckUd1a7yAnOxKdMghiPqHQvdzrNJjgcK6WNZWAFxC6LBBCIoiQXCOn7Ap5wqWEGyz7AWlNINPnoBDNrEhnBDuFckZml7l2fp7I5vjugZ6hP1Z1gHKHeRrIlBseWCcst0uOIoFeHgnN18Jh49WdNmbwo4bSrVOmFzWU/0Y+VnrL3HeNYh2xPhpR0szSlBuqUCPjj9jnYD2d7JIYFj8b88k9Yk5dSXJER4l2K+FCEOJw5TFg3DQDEmrQsZPVKVOXSxs5kpjYYn/sfXnwjGJKN6MLlUnTMERdLcZqd/c8xLqQ0nMX0Cj+q27PH0dQAZcxVCLZ80nC+35Z8Ngf0RUxW94TTI74jkuePTkMNcAxCGuSEKSXBwPVkqCugzrc/wh3fY+ioyP+8PAkxZr+PXnulQgnma4O09UKF12LVKBTXdbOKeEodyExFSTylwK4kOno2FrXfQwJ40pMPrC5sLm6+6g3JrXrd7zGsz5KazkmP+TnTklTD/vWfG/EXdQwFc7Peg9GhR+hPETWcrG1/sTlky4gulC+XVBMPtore49fwG4nzhWLbX4BPzhwxPfS1XA36grUXD1PfMUwfN8vYNQrwG+3Kif+V/Ul78zL3+Y4wKv8RN4qwoeL5QWMX3Q/mfaX+GL+HL/ublV6txBzhUeViapv/wGq925n). 

However I can't reproduce your issue @BHerbulot would you mind providing us with similar example? :)

**Comment by BHerbulot** [comment]
Sorry i forget to repost but the problem was fixed by himself. I guess an update fixed the problem.
